src/screens/Platform/CatalogCTA.js

```diff
@@ -0,0 +1,103 @@
+// @flow
+import React from "react";
+import { StyleSheet, View, TouchableOpacity } from "react-native";
+import { useTheme } from "@react-navigation/native";
+
+import LText from "../../components/LText";
+import IconExternalLink from "../../icons/ExternalLink";
+
+type CatalogCTAType = "secondary" | "dashed";
+
+type Props = {
+  type: CatalogCTAType,
+  children: React$Node,
+  title: React$Node,
+  onPress: Function,
+  Icon: React$ComponentType<{ size: number }>,
+  style: ReactNative$StyleProp,
+};
+
+const CatalogCTA = ({
+  type = "secondary",
+  children: description,
+  title,
+  Icon,
+  onPress,
+  style,
+}: Props) => {
+  const { colors } = useTheme();
+  const styles = getStyles(colors, type);
+
+  return (
+    <TouchableOpacity onPress={() => onPress()}>
+      <View style={[styles.root, style]}>
+        <View style={styles.content}>
+          <View style={styles.head}>
+            {Icon && (
+              <View style={styles.iconContainer}>
+                <Icon size={18} color={styles.title.color} />
+              </View>
+            )}
+            <LText style={styles.title} semiBold>
+              {title}
+            </LText>
+          </View>
+          <LText style={styles.description}>{description}</LText>
+        </View>
+        <IconExternalLink size={18} color={colors.smoke} />
+      </View>
+    </TouchableOpacity>
+  );
+};
+
+const getStyles = (colors, type) =>
```

Comment:
does this work in the end? if you switch theme and go back does it change here too?